tests/system/requests/test_upload.py

```diff
@@ -20,7 +20,10 @@
 import urllib.parse
 
 import pytest  # type: ignore
-import mock
+try:
```

Comment:
Thank you for doing this! I'm pleased to say we don't need the try/except anymore, as we have a separate Python 2.7 branch and so all supported versions of the client library should support unittest.mock. Just the bare `from unittest import mock` should be enough.

Reply:
Thanks! Will update it now.